Ports/libicu/package.sh

```diff
@@ -1,20 +1,20 @@
 #!/usr/bin/env -S bash ../.port_include.sh
-port=libicu
-version=69.1
-useconfigure=true
-use_fresh_config_sub=true
-workdir=icu/source
+port='libicu'
+version='69.1'
+useconfigure='true'
+use_fresh_config_sub='true'
+workdir='icu/source'
 files=(
     "https://github.com/unicode-org/icu/releases/download/release-${version//./-}/icu4c-${version//./_}-src.tgz 4cba7b7acd1d3c42c44bb0c14be6637098c7faf2b330ce876bc5f3b915d09745"
 )
 
 configure() {
     host_env
     run mkdir -p ../host-build
-    run sh -c "cd ../host-build && ../source/configure && make ${makeopts[@]}"
+    run sh -c "cd ../host-build && ../source/configure && make ${makeopts[*]}"
```

Comment:
Does this actually make any difference in practice?

Reply:
This removes a shellcheck warning: `Argument mixes string and array. Use * or separate argument`. Shellcheck [SC2145](https://www.shellcheck.net/wiki/SC2145).